signac/contrib/project.py

```diff
@@ -618,9 +618,9 @@ def _find_job_ids(self, filter=None, doc_filter=None, index=None):
         if filter is None and doc_filter is None and index is None:
             return list(self._job_dirs())
         if index is None:
-            filter = dict(parse_filter(filter, 'sp'))
+            filter = dict(parse_filter(filter, 'sp.'))
```

Comment:
I think we should use the explicit keyword argument here, because otherwise it is hard to follow without knowing the function signature of `parse_filter()`.
```suggestion
            filter = dict(parse_filter(filter, prefix='sp.'))
```
I'd actually go even a step further and use an even more explicit code structure:
```suggestion
            filter = dict(parse_filter(add_prefix('sp.', filter)))
```
Here I also switched the argument order for `add_prefix()`, because that would logically make more sense to me.

_Any changes proposed above would of course also apply to the other spots in the code with the same pattern._